abe/helper_functions/email_helpers.py

```diff
@@ -159,16 +165,13 @@ def cal_to_event(cal):
         reply_email(sender, new_event_dict)
         return new_event_dict, 201
 
+
 def smtp_connect():
     """ Connects to the smtp server
     :return: server instance, gmail to send
     """
-    try:
-        gmail = os.environ['ABE_EMAIL'] # should be 'abe.at.olin@gmail.com'
-        gpass = os.environ['ABE_PASS'] # should be 'abe@olin'
-    except:
-        gmail = 'abe.at.olin@gmail.com'
-        gpass = 'abe@olin'
+    gmail = ABE_EMAIL
+    gpass = ABE_PASS
     try:
```

Comment:
This `try` ends with a bare `except`, that prints the error but then returns normally. What should actually actual happen if there's an exception? Should the program really try to continue, or should it terminate? If the former, is there a more specific exception that it can catch?

Reply:
If it tries to run without having access to the email, I think it will struggle every time it tries to scrape (so once a minute). This seems less than ideal, but because of the way it is set up, if it stops trying to scrape and quits that loop as soon as it hits a KeyError, everything else should be able to function as normal. I believe it needs to be os.environ[]